doozer

```diff
@@ -229,6 +253,11 @@ def images_update_dockerfile(runtime, stream, version, release, repo_type, messa
     - If '+', the current release will be bumped.
     - Else, the literal value will be set in the Dockerfile.
     """
+
+    if runtime.local:
+        yellow_print('images:update-dockerfile is not valid when using --local, use images:rebase instead')
```

Comment:
I love that more and more you're using color coding for informational messages now. That extra lil' bit of context is so helpful for at-a-glance checking

Reply:
:D   Note that I added `color_print()` which lets you give it other ANSI color names (I used this in `images:show-tree`